pkg/cmd/roachtest/cluster.go

```diff
@@ -1806,7 +1806,25 @@ func (c *cluster) Install(
 
 // RunE runs a command on the specified node, returning an error.
 func (c *cluster) RunE(ctx context.Context, node nodeListOption, args ...string) error {
-	return c.RunL(ctx, c.l, node, args...)
+	cmdString := strings.Join(args, " ")
+	logFile := fmt.Sprintf(`run_%s_%s_%s`,
+		timeutil.Now().Format(`15:04:05.000`),
+		c.makeNodes(node),
+		regexp.MustCompile(`[^a-z0-9_]+`).ReplaceAllString(cmdString, "_"),
+	)
+	// NB: we set no prefix because it's only going to a file anyway.
+	l, err := c.l.ChildLogger(logFile, quietStderr, quietStdout)
+	if err != nil {
+		return c.RunL(ctx, c.l, node, args...)
```

Comment:
I should probably just return the error here. Is it ever expected that ChildLogger fails?